addon/TelemetryRappor.jsm

```diff
@@ -14,10 +14,29 @@ const PREF_RAPPOR_PATH = "toolkit.telemetry.rappor.";
 const PREF_RAPPOR_SECRET = PREF_RAPPOR_PATH + "secret";
 
 Cu.import("resource://gre/modules/Console.jsm");
+Cu.import("resource://gre/modules/Services.jsm");
+Cu.importGlobalProperties(['crypto']);
 
 const console = new ConsoleAPI({prefix: "shield-study-rappor"});
```

Comment:
This seems [sufficiently stable](http://searchfox.org/mozilla-central/source/toolkit/modules/Log.jsm) (it's been in dev. since 2012). Let's just use it, we can always fall back.